drivers/video/mt9m114.c

```diff
@@ -381,6 +383,46 @@ static int mt9m114_set_output_format(const struct device *dev, int pixel_format)
 	return ret;
 }
 
+static int mt9m114_set_horizontal_mirror(const struct device *dev, int enable)
+{
+	int ret = 0;
+	uint16_t reg = 0;
+
+	ret = mt9m114_read_reg(dev, MT9M114_CAM_SENSOR_CTRL_READ_MODE, sizeof(reg), &reg);
+
+	if (ret < 0) {
+		return ret;
+	}
+
+	if (enable) {
+		reg |= BIT(0);
+	} else {
+		reg &= ~BIT(0);
+	}
+
+	return mt9m114_write_reg(dev, MT9M114_CAM_SENSOR_CTRL_READ_MODE, sizeof(reg), &reg);
+}
+
+static int mt9m114_set_vertical_flip(const struct device *dev, int enable)
```

Comment:
Unless I missed something the two functions are after all essentially duplicated? Maybe make it a common set_orientation function?

Reply:
You are right, I used two different functions like other camera driver does, but in this case, the functions may be the same adding just the bit that you want to change. Thus, I will merge them into just one function